karate-core/src/main/java/com/intuit/karate/core/ExceptionUtils.java

```diff
@@ -0,0 +1,23 @@
+package com.intuit.karate.core;
+
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+public class ExceptionUtils {
```

Comment:
I think I recently added this to `StringUtils`

Reply:
Will change it. Still have some code for a factory for the Driver